qt_dotgraph/test/dot_to_qt_test.py

```diff
@@ -53,7 +53,7 @@ def check_x_server():
 
 class DotToQtGeneratorTest(unittest.TestCase):
 
-    DOT_CODE = '''
+    DOT_CODE = r'''
```

Comment:
There is a '\N' in the string and that causes issues with Python3 since it tries to parse that as a unicode control character. To avoid that, it needs to be specified to be a raw string.
```
$ python3
>>> print('\N')
  File "<stdin>", line 1
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 0-1: malformed \N character escape
>>> print(r'\N')
\N
```